fe/fe-core/src/main/java/com/starrocks/planner/RuntimeFilterDescription.java

```diff
@@ -30,6 +30,8 @@
     private int equalCount;
     private int crossExchangeNodeTimes;
 
+    // This flag is only for UT, we can push down rf on best effort.
+    private static boolean alwaysCanProbeUse = false;
```

Comment:
Why special for UT?

Reply:
Because to push down RF, cardinality and selectivity have to be considered in the real scenario. 

But in UT, we just want to make sure RF can be pushed down to the lowest level. We can ignore cardinality and selectivity. And that flag `alwaysCanProbeUse` is for that: push down RF at best effort and ignore cardinality/selectivity.

Reply:
Remove this flag.